packages/@react-spectrum/actionbar/docs/ActionBar.mdx

```diff
@@ -0,0 +1,274 @@
+
+```tsx example
+function DynamicActionBar() {
+  let barItems = [
+    {key: 'edit', label: 'Edit'},
+    {key: 'copy', label: 'Copy'},
+    {key: 'delete', label: 'Delete'}
+  ];
+
+  return (
+    <ActionBarContainer>
+      <ActionBar
+        items={barItems}
+        selectedItemCount={3}
+        isEmphasized>
+        {item => (
+          <Item key={item.key}>
+            {item.label}
+          </Item>
+        )}
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+
+<DynamicActionBar />
+```
+
+### Layout
+ActionBarContainer accepts an ActionBar alongside any renderable children. It accepts the typical selection of [style props](#props), allowing you to customize its layout.
+The example below applies a static height on the ActionBarContainer instead of matching the height of the TableView.
+
+```tsx example
+import {Cell, Column, Row, TableBody, TableHeader, TableView} from '@react-spectrum/table';
+import {Text} from '@react-spectrum/text';
+
+function ActionBarTable(props) {
+  // Same bar items as example above...
+  ///- begin collapse -///
+  let barItems = [
+    {key: 'edit', label: 'Edit'},
+    {key: 'copy', label: 'Copy'},
+    {key: 'delete', label: 'Delete'}
+  ];
+  ///- end collapse -///
+  let columns = [
+    {name: 'Name', uid: 'name'},
+    {name: 'Type', uid: 'type'},
+    {name: 'Level', uid: 'level'}
+  ];
+
+  let rows = [
+    {id: 1, name: 'Charizard', type: 'Fire, Flying', level: '67'},
+    {id: 2, name: 'Blastoise', type: 'Water', level: '56'},
+    {id: 3, name: 'Venusaur', type: 'Grass, Poison', level: '83'},
+    {id: 4, name: 'Pikachu', type: 'Electric', level: '100'}
+  ];
+
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([2]));
+  return (
+    <ActionBarContainer height="size-5000">
+      <TableView aria-label="Table with controlled selection" selectionMode="multiple" selectedKeys={selectedKeys} onSelectionChange={setSelectedKeys} height="size-3000">
+        <TableHeader columns={columns}>
+          {column => (
+            <Column key={column.uid} align={column.uid === 'level' ? 'end' : 'start'}>
+              {column.name}
+            </Column>
+          )}
+        </TableHeader>
+        <TableBody items={rows}>
+          {item => (
+            <Row>
+              {columnKey => <Cell>{item[columnKey]}</Cell>}
+            </Row>
+          )}
+        </TableBody>
+      </TableView>
+      <ActionBar
+        items={barItems}
+        isEmphasized
+        selectedItemCount={selectedKeys === 'all' ? selectedKeys : selectedKeys.size}>
+        {item => (
+          <Item key={item.key}>
+            {item.label}
+          </Item>
+        )}
+      </ActionBar>
+    </ActionBarContainer>
+  );
+}
+```
+
+### Accessibility
+The contents of the ActionBar should follow the same accessibility guidelines as ActionGroup's items, see the [ActionGroup docs](./ActionGroup.html#accessibility) for more details.
```

Comment:
Should we have an example that uses icons and aria-label? I don't see that elsewhere on this page. The example in ActionGroup does this. I'm wondering if we need to be explicate or repeat ourselves?

Reply:
I figure users would look at the ActionGroup docs for that, don't wanna repeat too much if at all possible IMO.